consumer/src/test/kotlin/integrations/kafka/KafkaEventSinkBase.kt

```diff
@@ -64,14 +64,15 @@ open class KafkaEventSinkBase {
     fun setUp() {
         graphDatabaseBuilder = ImpermanentDbmsRule()
                 .setConfig("kafka.bootstrap.servers", KafkaEventSinkSuiteIT.kafka.bootstrapServers)
-                .setConfig("kafka.zookeeper.connect", KafkaEventSinkSuiteIT.kafka.envMap["KAFKA_ZOOKEEPER_CONNECT"] ?: "")
+                .setConfig("kafka.zookeeper.connect", KafkaEventSinkSuiteIT.kafka.envMap["KAFKA_ZOOKEEPER_CONNECT"] ?: "localhost:2181" ?: "")
```

Comment:
make sure to document what the default assumption is, otherwise this will be populated, users will get "can't connect message" and will file a ticket.  :)